File: easyevent_frontend/easyevent/src/models/dto/ListUserResponse.ts
export class ListUserResponse {
    id: number;
    name: string;
    email: string;
    profilePicture: string | null;
    active: boolean;
    banned: boolean;
    roles: string[];

    constructor(id: number, name: string, email: string, profilePicture: string|null, active: boolean, banned: boolean, roles: string[]) {
        this.id = id;
        this.name = name;
        this.email = email;
        this.profilePicture = profilePicture;
        this.active = active;
        this.banned = banned;
        this.roles = roles;
    }
}

export default ListUserResponse;

File: easyevent_backend/src/main/java/com/benny1611/easyevent/dto/ListUserResponse.java
package com.benny1611.easyevent.dto;

import lombok.Data;

import java.util.List;

@Data
public class ListUserResponse {
    private Long id;
    private String name;
    private String email;
    private String profilePicture;
    private boolean active;
    private boolean isBanned;
    private List<String> roles;
}
